### DISCUSSION TITLE
UWAppDev Club Website

### DISCUSSION THREAD

**Discussion by SilliJelli** [UWAppDev/community#18] [Projects]
## Overview

The [current website of UWAppDev](https://uwapp.dev) is in need of redesign & reimplementation. The goal for now is to implement a set of visuals along the lines of [this design](https://drive.google.com/drive/folders/1fiqzV3DmqyohSwRDVnX2NYMHASwxe6JT?usp=sharing) (but we don't have to follow this!).
We are opening up this project for our community to contribute so that interested people may gain some Flutter/web dev experience!

## How you can contribute

- Webpage content planning/organization
- Write code to implement the website
- Design website visuals
- Manage project issues, contributors, balance the workload, and check in on progress
 
## Expectations

- Everyone
  - Attend meetings (as many as you can) and meet deadlines (if we do set deadlines)
- Coder
  - Have some knowledge of web dev, or is willing to learn.
- Designer
  - Have an eye for aesthetics
  - Have access to at least one graphic editing/web design software (e.g. Adobe Photoshop, Adobe Dreamweaver, Autodesk SketchBook, Procreate) for the duration of this project
    - If you are interested in design but do not have access to these softwares, please let me (@SilliJelli) know and we can figure out some alternative solutions together

## Join the Project!

- Contact @SilliJelli (IntelliJelly#0973) to get access to the #website channel on discord
- Meeting location: Online or HUB 337
- Meeting times: TBD, Sat. 4-6 pm for the time being
- GitHub pages: [https://github.com/UWAppDev/uwappdev.github.io](https://github.com/UWAppDev/uwappdev.github.io)

**Comment by ApolloZhu** [comment]
Update: dropped references to Flutter as we found it to be an unideal implementation for static hosting.

**Comment by ApolloZhu** [comment]
For your reference, a (probably outdated) internal document from November 30th, 2020 suggests the website should have the following features:

- Home
   - Some animation in the BG & join our discord button
   - News & updates (more recent ones for the home page)
   - Smol embedded calendar?
- About Us
   - Our mission
   - Our members & mentors (???
   - Maybe put here some projects we did in the past
- News & Events
   - Upcoming events! With a embedded google calendar
   - Maybe also post here when a new tutorial is published
   - News & updates (more detailed for this page)
- Community - jumps to our blog post domain, https://uwdev.app
- Resources
   - past tutorials
   - DSC stuff (Feel free to grab archived materials from DSC notion)
- Socials, just a dropdown bar, not a whole new page / or icons at the bottom
